addons/calendar/models/calendar_attendee.py

```diff
@@ -119,15 +119,11 @@ def _send_mail_to_attendees(self, mail_template, force_send=False):
 
         # If the mail template has attachments, prepare copies for each attendee (to be added to each attendee's mail)
         if mail_template.attachment_ids:
-
-            # Setting res_model to ensure attachments are linked to the msg (otherwise only internal users are allowed link attachments)
-            attachments_values = [a.copy_data({'res_id': 0, 'res_model': 'mail.compose.message'})[0] for a in mail_template.attachment_ids]
-            attachments_values *= len(self)
-            attendee_attachment_ids = self.env['ir.attachment'].create(attachments_values).ids
-
-            # Map attendees to their respective attachments
-            template_attachment_count = len(mail_template.attachment_ids)
-            attendee_id_attachment_id_map = dict(zip(self.ids, split_every(template_attachment_count, attendee_attachment_ids, list)))
+            attendee_id_attachment_id_map = {
+                # Setting res_model to ensure attachments are linked to the msg (otherwise only internal users are allowed link attachments)
+                attendee_id: [a.copy({'res_id': 0, 'res_model': 'mail.compose.message'}).id for a in mail_template.attachment_ids]
```

Comment:
Hey Mattis :wave:
From what I understood in the code, we are copying the same attachments info for each attendee, so I think we can move this directly to some lines below (i.e. not creating a map):
```py
# Add template attachments copies to the attendee's email, if available
attachment_ids = attendee_id_attachment_id_map[attendee.id] if mail_template.attachment_ids else []```

What do you think?